core/controllers/feedback.py

```diff
@@ -284,6 +295,23 @@ def post(self):
             exploration_rights.owner_ids)
 
 
+class ReportEmailHandler(base.BaseHandler):
+    """Handler task of sending report emails to moderators"""
+
+    def post(self):
+        payload = json.loads(self.request.body)
+        exploration_id = payload['exploration_id']
+        report_type = payload['report_type']
```

Comment:
What do you think about, instead, having a free-form text field where users can type what the issue is? We can then add some select buttons to cover the more common cases, but this would allow them to tell us about things that we hadn't thought to include beforehand. I.e., a multiple-choice field with an "Other" option.

If so, perhaps this should be report_text instead.